aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -195,7 +195,29 @@ bool is_nonzero(const Tensor& self) {
 
 namespace {
 
-static Tensor wrapped_scalar_tensor(
+at::Tensor scalar_to_tensor_default_dtype(
```

Comment:
This function must still deserve some kind of comment, right? Even if it's "DO NOT USE THIS".

Reply:
oh, I didn't do that because it's just an implementation detail of wrapped_scalar_tensor below -- I'll make a note of that.